src/main/java/com/odysseusinc/arachne/datanode/service/impl/UserServiceImpl.java

```diff
@@ -85,6 +85,14 @@
     @Autowired
     private ApplicationEventPublisher eventPublisher;
 
+    @Override
+    public User create(User user) {
+
+        user.setEnabled(true);
```

Comment:
WHy do you ignore firstname, lastname, login, etc? I mean, they shouldn't be required (btw, what will be shown in navbar if names are empty?) but if they are presented - why ignore?

Reply:
Relying on the authenticator. One should fulfill this fields according to configuration.

Reply:
Ah, ok, note for myself: those a prefilled by converter